driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  webpack(config) {
+    config.experiments ??= {}
+
+    config.experiments.asyncWebAssembly = true
```

Comment:
```suggestion
    // https://webpack.js.org/configuration/experiments/#experiments
    config.experiments.asyncWebAssembly = true
```
It's nice to link the docs I think